src/openrct2-ui/windows/ShortcutKeys.cpp

```diff
@@ -302,11 +304,11 @@ static void window_shortcut_scrollmousedown(rct_window* w, int32_t scrollIndex,
         return;
 
     // Is this a separator?
-    if (ShortcutList[selected_item].ShortcutId == SHORTCUT_UNDEFINED)
+    if (ShortcutList[selected_item].ShortcutId == Input::Shortcut::Undefined)
         return;
 
     auto& shortcut = ShortcutList[selected_item];
-    window_shortcut_change_open(shortcut.ShortcutId, shortcut.StringId);
+    window_shortcut_change_open(static_cast<int32_t>(shortcut.ShortcutId), shortcut.StringId);
```

Comment:
change window_shortcut_change_open to take the correct type. Change gKeyboardShortcutChangeId to use the new type